src/main/java/net/snowflake/client/jdbc/DefaultSFConnectionHandler.java

```diff
@@ -76,7 +76,7 @@ public boolean supportsAsyncQuery() {
 
   @Override
   public void initializeConnection(String url, Properties info) throws SQLException {
-    initialize(conStr);
+    initialize(conStr, LoginInfoDTO.SF_JDBC_APP_ID, SnowflakeDriver.implementVersion);
```

Comment:
Do we really need these change here? Why can't we just overwrite them from snowpark?

Reply:
For more code reuse, please take a look at my PR here: https://github.com/snowflakedb/thundersnow/pull/421
Many variables and methods in this default handler are private, so I cannot access them even after inheriting this class. Changing the apis for a few private method here enables us to directly call those apis, while not affecting anything public. 
If we don't make this change, I'll have to almost copy the entire handler class.